src/runner/api/itemRunner.js

```diff
@@ -272,9 +272,7 @@ var itemRunnerFactory = function itemRunnerFactory(providerName, data, options)
             }
 
             if (flow.init.done === false) {
-                flow.init.pending.push(function() {
-                    this.render(elt, options);
-                });
+                flow.init.pending.push(() => this.render(elt, options));
```

Comment:
Is it safe? Usually `this` is different in function expression and arrow func

Reply:
Because in many cases you change `self` to `this` and use arrow func and it is okay but here already was `this` so I doubt

Reply:
please check, all fow function were called bound to `this`   (ie. https://github.com/oat-sa/tao-item-runner-fe/pull/23/files#diff-488a3a2a9d972319d0fda4595e363d1cL162)